js/src/index.js

```diff
@@ -6,10 +6,11 @@ import * as BlueprintSelect from '@blueprintjs/select';
 import * as Blueprint from '@blueprintjs/core';
 
 import * as Inputs from './inputs.jsx';
-import * as Suggests from './suggest.js';
-import * as Selects from './select.js';
-import * as MultiSelect from './multiselect.js';
-import * as MultiSlider from './multiSlider.js';
+import Suggest from './suggest.js';
+import Select from './select.js';
+import MultiSelect from './multiselect.js';
+import MultiSlider from './multiSlider.js';
+import Tree from './tree.js';
```

Comment:
I believe there's no need for the the extensions, the code below should work

```
import * as Inputs from './inputs';
import Suggest from './suggest';
import Select from './select';
import MultiSelect from './multiselect';
import MultiSlider from './multiSlider';
import Tree from './tree';
```

Reply:
Extensions removed :+1: